cli/ops/compiler.rs

```diff
@@ -69,7 +69,11 @@ fn op_fetch_source_files(
     None
   };
 
-  let global_state = state.borrow().global_state.clone();
+  let s = state.borrow();
+  let global_state = s.global_state.clone();
+  let permissions = s.permissions.clone();
+  let perms_ = permissions.clone();
+  drop(s);
```

Comment:
Why drop(s) ? Can you not just do `state.borrow().global_state.permissions.clone()` ?

Reply:
I'm cloning two different fields; so not to write `state.borrow()` multiple times I'm borrowing, cloning what's needed and dropping the borrow.

Reply:
Does it not compile fine if you remove the drop?

Reply:
It does, removed